eox_nelp/api_clients/certificates.py

```diff
@@ -49,7 +49,7 @@ def create_external_certificate(self, certificate_data):
         path = "Certificates"
         user = certificate_data["user"]
         payload = {
-            "reference_id": certificate_data["id"],
+            "reference_id": f"openedx-lms-cert-{certificate_data['id']}",
```

Comment:
I think that they could have an edx integration. Is something related futurex or nelp. Or just with the open is different than with edx. Also to remind the possibility of the existence of other openedx integration...?

Reply:
IMO the probability of that is almost null and probably when they have another integration you will have another job, but anyway I have already applied that